src/libpgagroal/configuration.c

```diff
@@ -570,6 +573,13 @@ pgagroal_read_configuration(void* shm, char* filename, bool emitWarnings)
                      unknown = true;
                   }
                }
+               else if (key_in_section("update_process_title", section, key, true, &unknown))
+               {
+                  if (as_update_process_title(value, &config->update_process_title, UPDATE_PROCESS_TITLE_STRICT))
```

Comment:
_VERBOSE, right ?

Reply:
It sounds like the best solution is to stay on `_STRICT` in this case then.

I'll await your decision :)

Reply:
After reasoning a bit, I decided to put `_VERBOSE` as the default even when the configuration is not understood. Seems to me coherency is better here.